monkey/monkey_island/cc/models/monkey.py

```diff
@@ -88,17 +98,17 @@ def get_os(self):
             os = "windows"
         return os
 
-    @staticmethod
     @ring.lru()
```

Comment:
If you only need class version of lru_cache, [methodtools](https://pypi.org/project/methodtools/) can be more efficient.

```python
@methodtools.lru_cache()
@staticmethod
def get_label_by_id(...):
    ...
```

Reply:
We also use deletion by key, that's why we need `ring`. 
https://github.com/guardicore/monkey/blob/dcbe7b1ee0264089b3e54e14704aa3ae3bc7d072/monkey/monkey_island/cc/models/monkey.py#L127